source/templates.tex

```diff
@@ -1280,7 +1280,7 @@
 
 struct A {
   constexpr A(const char*) {}
-  auto operator<=>(A, A) = default;
+  friend auto operator<=>(const A&, const A&) = default;
```

Comment:
This change is good; the =default is the definition, so "auto" works.

Reply:
Yes, this example wouldn't work without the `=default`, because the operator needs to be structural.